src/design/plone/contenttypes/restapi/services/types/get.py

```diff
@@ -149,15 +140,29 @@ def reply(self):
                             "geolocation", interface=IGeolocationDefaults
                         )
                     )
+
         # be careful: result could be dict or list. If list it will not
         # contains title. And this is ok for us.
         pt = self.request.PATH_INFO.split("/")[-1]
-        if "title" in result and pt == "Venue":
-            result = self.customize_venue_schema(result)
-
         if "title" in result and pt == "Persona":
             result = self.customize_persona_schema(result)
-
-        if "title" in result and pt == "Event":
-            result = self.customize_evento_schema(result)
         return result
+
+    def reorder_fieldsets(self, original):
+        pt = self.request.PATH_INFO.split("/")[-1]
```

Comment:
Non mi ricordo se l'info da qui ce l'hai sia in edit che in add... mi pare di si e forse l'avevo pure fatto io...

Reply:
boh, ho copiato da quello che c'era già

Reply:
ok, dovrebbe andare.. quello che avevo fatto l'ho verificato sia in add che edit